python/paddle/v2/reader/decorator.py

```diff
@@ -298,19 +289,27 @@ def order_handle_worker(in_queue, out_queue, mapper, out_order):
         in_queue.put(end)
         out_queue.put(end)
 
-    # start several handle_workers
-    target = order_handle_worker if order else handle_worker
-    args = (in_queue, out_queue, mapper, out_order) if order else (
-        in_queue, out_queue, mapper)
-    workers = []
-    for i in xrange(process_num):
-        worker = Thread(target=target, args=args)
-        worker.daemon = True
-        workers.append(worker)
-    for w in workers:
-        w.start()
-
     def xreader():
```

Comment:
'xmap_readers' is like 'map_readers' in decorator.py. They are truly decorators.